api/src/Microsoft.Azure.IIoT.Api/src/Publisher/Extensions/ModelExtensions.cs

```diff
@@ -1413,7 +1413,7 @@ public static class ModelExtensions {
                 PublishedVariableNodeId = model.PublishedVariableNodeId,
                 BrowsePath = model.BrowsePath,
                 Attribute = model.Attribute,
-                DataChangeFilter = (OpcUa.Publisher.Models.DataChangeTriggerType?)model.DataChangeFilter,
+                DataChangeTrigger = (OpcUa.Publisher.Models.DataChangeTriggerType?)model.DataChangeFilter,
```

Comment:
Here is another of these, can you change the DataChangeFilter which is a DataChangeTrigger to be named DataChangeTrigger?